packages/next-swc/crates/next-api/src/project.rs

```diff
@@ -575,3 +575,18 @@ impl Project {
             .keys_in_path(self.client_root()))
     }
 }
+
+#[turbo_tasks::function]
+async fn all_assets_from_entries_operation_inner(
+    operation: Vc<OutputAssetsOperation>,
+) -> Result<Vc<OutputAssets>> {
+    let assets = *operation.await?;
+    Vc::connect(assets);
```

Comment:
Can you explain why we need to reconnect this? Why doesn't waiting on the result of `all_assets_from_entries` handle this already?

Reply:
Only calling functions makes them active. Awaiting doesn't do that. Usually that's fine, since the Vc is from a called function connected to a parent task, so it's active for other reasons anyway. But in this case we need to have the whole graph connected below this one function, to allow reconnecting it later again.